app/views/admin/conferences/new.html.haml

```diff
@@ -1,6 +1,6 @@
 .row
   .col-md-8
-    = semantic_form_for(@conference, :url => admin_conferences_path) do |f|
+    = semantic_form_for(@conference, url: admin_conferences_path) do |f|
```

Comment:
What about prepend in https://github.com/divyanshumehta/osem/blob/cd7cf2a9710744b4c4c8db91c5afd51f1a12b226/app/views/admin/conferences/new.html.haml#L8